src/app/admin/components/reports/reports.directive.js

```diff
@@ -406,10 +405,6 @@
                         statusActivity.details = statusChange;
                         output.status.push(statusActivity);
 
-                        activity.details.push('<span class="bg-danger"><strong>' + statusChange + '</strong></span>');
```

Comment:
Why aren't you pushing the statusChange to the regular activity field? It's currently there, so it should stay there after removing the "questionable" flag / display tweak